### ISSUE TITLE
Flow color when adding new Element in between changes

### ISSUE THREAD

**Issue by mtrana** [bpmn-io/bpmn-js#1892]
### Describe the Bug

when I have two Elements and a custom color flow between them (for example red flow) and I add between them another Element the color of the second half of the flow changes to black.

### Steps to Reproduce

1. Create two tasks (Task 1 and Task 2) and add a red flow between them.

![image](https://user-images.githubusercontent.com/131269449/233085615-05117b64-0483-4ada-b936-05fa60d2f39f.png)

2. Create a new empty task and add it between the two previous tasks. As you can see the color of the second flow turns black.

![image](https://user-images.githubusercontent.com/131269449/233085888-bcd30994-369e-4ec8-984f-e1adf3ea213c.png)

### Expected Behavior

It is expected that that both flows have the same color in this case red. 

![image](https://user-images.githubusercontent.com/131269449/233086361-c7714b38-4ac1-41d8-a841-016eb97a8740.png)

### Environment

 - Browser: [e.g. IE 11, Chrome 69]
 - OS: [e.g. Windows 7]
 - Library version: [e.g. 2.0.0]

**Comment by nikku** [comment]
Agreed. The flow splits and hence both sides should keep the same color.

Happy to take a contribution to fix this. The relevant place would be [`DropOnFlowBehavior`](https://github.com/bpmn-io/bpmn-js/blob/develop/lib/features/modeling/behavior/DropOnFlowBehavior.js#L102).

**Comment by mtrana** [comment]
`} else {
        outgoingConnection = modeling.connect(
          shape, target, { type: targetFlow.type, waypoints: waypointsAfter}
        );
        modeling.setColor(outgoingConnection, {stroke: targetFlow.di.stroke});
        modeling.updateProperties(outgoingConnection, {flowType: targetFlow.di.flowType});
      }`

I achieved to copy the right color on the second flow but my custom property 'flowtype'  cannot be copied. I cannot find under which attribute it is saved. Do you have any idea where custom attributes are being saved? targetFlow.di.flowType and targetFlow.flowType dont exist.

**Comment by philippfromme** [comment]
> I achieved to copy the right color on the second flow but my custom property 'flowtype' cannot be copied. I cannot find under which attribute it is saved. Do you have any idea where custom attributes are being saved? targetFlow.di.flowType and targetFlow.flowType dont exist.

This is not about colors though, is it? Your custom properties usually exist on the business object, not on the DI.

**Comment by mtrana** [comment]
In my opinion all properties of the incoming flow should be copied to the outgoing flow - including the flow color as well as custom properties. I have tried to find my custom properties under the 'targetFlow.businessObject.customAttribute' but it's undefined. As soon as I drop an element on the flow and the flow splits I am not able to find my custom attributes anymore, neither for incoming or outcoming flow. However when I download the bpmn file I can see the custom attribute for the incoming flow but not for the outgoing one.
Any idea why that happens and how to fix that?
